File: tests/fixtures/java/01-obvious-noise/input.java
class UserService {
    User getUser(String id) {
        // Verificar si el usuario existe
        User user = db.findById(id);

        // Si no existe, retornar error
        if (user == null) {
            throw new NotFoundException();
        }

        // Devolver el usuario
        return user;
    }
}

File: tests/fixtures/go/01-obvious-noise/input.go
package user

func GetUser(id string) (*User, error) {
	// Verificar si el usuario existe
	user, err := db.FindByID(id)
	if err != nil {
		return nil, err
	}

	// Si no existe, retornar error
	if user == nil {
		return nil, ErrNotFound
	}

	// Devolver el usuario
	return user, nil
}
